### DISCUSSION TITLE
Changing title bar style in tabbed windows

### DISCUSSION THREAD

**Discussion by arash28134** [i3/i3#5607] [Q&A] [answered]
How can I change the style of title bars in tabbed windows like this so the bottom snaps to the focused window?

(Removing this gap between title bar and focused window)
![image](https://github.com/i3/i3/assets/74412308/9c784ba7-54ab-4c50-a1d6-910f6109a745)

**Comment by okraits** [comment, marked as answer]
I guess this gap is caused by your compositor. i3 itself doesn't draw rounded window borders and thus no such gap (if you and me mean the same gap). You might need to have a look at the settings of the compositor.

**Reply by arash28134** [reply]
Yeah you were right. my bad.
I was using [Picom ](https://github.com/yshui/picom)if anyone is curious.
